Tests/NIOTests/SelectorTest.swift

```diff
@@ -358,7 +358,7 @@ class SelectorTest: XCTestCase {
                         return channel
                     }
                 }
-        }.wait().forEach { return try! $0.wait() } as Void)
+        }.wait().forEach { _ = try! $0.wait() } as Void)
```

Comment:
Rather than just suppress the return value, why not wrap this in a `XCTAssertNoThrow` and remove the `try!`? That way we have `.forEach { XCTAssertNoThrow(try $0.wait) }`, which would be a bit better.

Reply:
sure why not...